recurly/errors.py

```diff
@@ -256,5 +258,5 @@ def new_status_error(xml_response):
             return UnexpectedStatusError(status, xml_response)
         return new_status_error
 
-
-__all__ = [x.__name__ for x in error_classes.values()]
+other_errors = [ConfigurationError]
+__all__ = [x.__name__ for x in list(error_classes.values()) + other_errors]
```

Comment:
I needed the explicit list(conversion) for python 3 support. Perhaps there is a cleaner way.